macros/base/start_print.cfg

```
@@ -33,6 +34,7 @@ gcode:
     SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=material VALUE='"{MATERIAL}"'
     SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=fl_size VALUE='"{FL_SIZE}"'
     SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=bed_mesh_profile VALUE='"{BED_MESH_PROFILE}"'
+    SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=force_mesh VALUE='"{FORCE_MESH}"'
```

In both case (integer or boolean use), this needs to be changed here to avoid having a "False" string evaluated as True due to how Jinja read strings.

```suggestion
    SET_GCODE_VARIABLE MACRO=START_PRINT VARIABLE=force_mesh VALUE={FORCE_MESH}
```